flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/sql/MatchRecognizeITCase.scala

```diff
@@ -543,10 +543,79 @@ class MatchRecognizeITCase extends StreamingWithStateTestBase {
     // We do not assert the proctime in the result, cause it is currently
```

Comment:
I guess this problem has not been solved with the new rich function?

Reply:
Unfortunately not, there is a separate issue for this: https://issues.apache.org/jira/browse/FLINK-10596